test/apps/iodemo/io_demo.cc

```diff
@@ -1563,29 +1560,28 @@ class DemoClient : public P2pDemoCommon {
     void disconnect_server(size_t server_index, const char *reason) {
         server_info_t& server_info = _server_info[server_index];
         bool disconnecting         = server_info.conn->is_disconnecting();
-        std::stringstream ss;
 
-        if (disconnecting) {
-            ss << "not ";
-        }
+        {
+            UcxLog log(LOG_PREFIX);
+            if (disconnecting) {
```

Comment:
inverse

Reply:
answered, this is correct, it was a bug before the change